src/WebJobs.Extensions.OpenAI.AISearch/AzureAISearchConfigOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenAI.AzureAISearch;
+
+/// <summary>
+/// Open AI Configuration Options used for reading host.json values.
+/// </summary>
+public class AzureAISearchConfigOptions
+{    
+    public string IsSemanticSearchEnabled { get; set; } = "IsSemanticSearchEnabled";
+
+    public string UseSemanticCaptions { get; set; } = "UseSemanticCaptions";
+
+    public string VectorSearchDimensions { get; set; } = "vectorSearchDimensions";
```

Comment:
Shouldn't all these values be set to `int` or `bool` values? Looking at `AzureAISearchProvider.cs`, it seems like you're attempting to parse them as such.

Reply:
updated the type